CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log for coreJSON Library
 
+## v3.0.2 (July 2021)
+- [#100](https://github.com/FreeRTOS/coreJSON/pull/100) Fix overflow in skipOneHexEscape().
```

Comment:
As there is a bugfix update, should this undergo a minor version update instead of a patch version change?